__tests__/app.test.js

```diff
@@ -0,0 +1,72 @@
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+const db = require("../db/connection");
+const app = require("../app");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api", () => {
+  describe("GET /topics", () => {
+    test("GET 200: endpoint responds with an array of topics", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          const { topics } = body;
+          expect(Array.isArray(topics)).toBe(true);
+          topics.forEach((topic) => {
```

Comment:
It would be useful here to check that the length of the array is what you would expect, to ensure all topics have been returned